llvm/lib/MC/DXContainerRootSignature.cpp

```diff
@@ -12,44 +12,59 @@
 using namespace llvm;
 using namespace llvm::mcdxbc;
 
+void setRewrite(raw_ostream &Stream, uint32_t &Offset) {
```

Comment:
I think these two functions are only used from within this cpp.  Should either be marked as static, or in an anonymous namespace, to make that clear.

(I'm not sure what LLVM's preference is between static / anonymous namespace)